src/include/daos_srv/vos_types.h

```diff
@@ -173,17 +176,33 @@ enum {
  * Returned entry of a VOS iterator
  */
 typedef struct {
-	/** Returned epoch. It is ignored for container iteration. */
-	daos_epoch_t		ie_epoch;
-	/** Returned earliest update epoch for a key */
-	daos_epoch_t		ie_earliest;
+	union {
+		/** Returned epoch. It is ignored for container iteration. */
+		daos_epoch_t			ie_epoch;
+		/** Return the DTX identifier. */
+		struct daos_tx_id		ie_xid;
```

Comment:
this will conflict with my patch though I imagine this will land first

Reply:
Yes, I noticed that, I will rebase this patch if yours landed firstly.